Formula/libtins.rb

```diff
@@ -16,19 +16,18 @@ class Libtins < Formula
   end
 
   depends_on "cmake" => :build
-  depends_on "openssl@1.1"
+  depends_on "openssl@3"
 
   uses_from_macos "libpcap"
 
   def install
-    mkdir "build" do
-      system "cmake", "..", *std_cmake_args, "-DLIBTINS_ENABLE_CXX11=1"
-      system "make", "install"
-      doc.install "examples"
```

Comment:
The examples are not built (even with default `LIBTINS_BUILD_EXAMPLES=ON`) as CMake has EXCLUDE_FROM_ALL. So, the bottle just has some useless files:
```console
❯ tree libtins/4.4/share/doc
libtins/4.4/share/doc
└── libtins
    └── examples
        ├── Makefile
        └── cmake_install.cmake
```

Although they can be built with `--target examples`, didn't seem like something we need to include in bottle and build/install may require fixing the RPATHs.